{
	"documenttitle": {
		"text": "Red Hat Security Advisory: Red Hat JBoss Data Virtualization 6.4.8.SP1 security update",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHSA-2022:0497"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2022-02-09T13:11:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2022-02-09T13:11:00Z",
		"CurrentReleaseDate": "2022-02-09T13:11:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.4.3",
			"Date": "2022-02-09T22:36:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nAn update is now available for Red Hat JBoss Data Virtualization.\n\nRed Hat Product Security has rated this update as having a security impact of Important. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nRed Hat JBoss Data Virtualization is a lean data integration solution that provides easy, real-time, and unified data access across disparate sources to multiple applications and users. JBoss Data Virtualization makes data spread across physically distinct systems - such as multiple databases, XML files, and even Hadoop systems - appear as a set of tables in a local database.\n\nThis Service Pack release of Red Hat JBoss Data Virtualization 6.4.8.SP1 (Service Pack 1) serves as a replacement for Red Hat JBoss Data Virtualization 6.4.8, and mitigates the impact of the log4j CVE's referenced in this document by removing the affected classes from the patch.\n\nNote: customers should update their EAP 6.4 installation with the corresponding security fixes that have been released for that (see RHSA-2022:0437 and https://access.redhat.com/site/solutions/625683)\n\nSecurity Fix(es):\n\n* log4j: deserialization of untrusted data in SocketServer (CVE-2019-17571)\n\n* log4j: SQL injection in Log4j 1.x when application is configured to use JDBCAppender (CVE-2022-23305)\n\n* log4j: Unsafe deserialization flaw in Chainsaw log viewer (CVE-2022-23307)\n\n* log4j: Remote code execution in Log4j 1.x when application is configured to use JMSAppender (CVE-2021-4104)\n\n* log4j: Remote code execution in Log4j 1.x when application is configured to use JMSSink (CVE-2022-23302)\n\n* log4j: improper validation of certificate with host mismatch in SMTP appender (CVE-2020-9488)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright Â© 2022 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Important",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHSA-2022:0497",
				"Description": "https://access.redhat.com/errata/RHSA-2022:0497"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/security/updates/classification/#important",
				"Description": "https://access.redhat.com/security/updates/classification/#important"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=data.services.platform&downloadType=securityPatches&version=6.4",
				"Description": "https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=data.services.platform&downloadType=securityPatches&version=6.4"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/documentation/en-us/red_hat_jboss_data_virtualization/6.4/html/release_notes/",
				"Description": "https://access.redhat.com/documentation/en-us/red_hat_jboss_data_virtualization/6.4/html/release_notes/"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "log4j: deserialization of untrusted data in SocketServer",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-20T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-17571",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:0497"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-17571",
						"Description": "CVE-2019-17571"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1785616",
						"Description": "bz#1785616: CVE-2019-17571 log4j: deserialization of untrusted data in SocketServer"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2019-12-20T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "2",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "log4j: improper validation of certificate with host mismatch in SMTP appender",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-04-25T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-9488",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:0497"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-9488",
						"Description": "CVE-2020-9488"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1831139",
						"Description": "bz#1831139: CVE-2020-9488 log4j: improper validation of certificate with host mismatch in SMTP appender"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2020-04-25T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "3",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "log4j: Remote code execution in Log4j 1.x when application is configured to use JMSAppender",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-12-10T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-4104",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:0497"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://github.com/apache/logging-log4j2/pull/608#issuecomment-990494126",
						"Description": "https://github.com/apache/logging-log4j2/pull/608#issuecomment-990494126"
					},
					{
						"URL": "https://github.com/apache/logging-log4j2/pull/608#issuecomment-991723301",
						"Description": "https://github.com/apache/logging-log4j2/pull/608#issuecomment-991723301"
					},
					{
						"URL": "https://lists.apache.org/thread/0x4zvtq92yggdgvwfgsftqrj4xx5w0nx",
						"Description": "https://lists.apache.org/thread/0x4zvtq92yggdgvwfgsftqrj4xx5w0nx"
					},
					{
						"URL": "https://www.openwall.com/lists/oss-security/2021/12/13/1",
						"Description": "https://www.openwall.com/lists/oss-security/2021/12/13/1"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-4104",
						"Description": "CVE-2021-4104"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2031667",
						"Description": "bz#2031667: CVE-2021-4104 log4j: Remote code execution in Log4j 1.x when application is configured to use JMSAppender"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-12-13T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "4",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "log4j: Remote code execution in Log4j 1.x when application is configured to use JMSSink",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-01-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-23302",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:0497"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.openwall.com/lists/oss-security/2022/01/18/3",
						"Description": "https://www.openwall.com/lists/oss-security/2022/01/18/3"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-23302",
						"Description": "CVE-2022-23302"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2041949",
						"Description": "bz#2041949: CVE-2022-23302 log4j: Remote code execution in Log4j 1.x when application is configured to use JMSSink"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-01-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "5",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "log4j: SQL injection in Log4j 1.x when application is configured to use JDBCAppender",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-01-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-23305",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:0497"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.openwall.com/lists/oss-security/2022/01/18/4",
						"Description": "https://www.openwall.com/lists/oss-security/2022/01/18/4"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-23305",
						"Description": "CVE-2022-23305"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2041959",
						"Description": "bz#2041959: CVE-2022-23305 log4j: SQL injection in Log4j 1.x when application is configured to use JDBCAppender"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2022-01-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "6",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "log4j: Unsafe deserialization flaw in Chainsaw log viewer",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-01-18T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-23307",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:0497"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.openwall.com/lists/oss-security/2022/01/18/5",
						"Description": "https://www.openwall.com/lists/oss-security/2022/01/18/5"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-23307",
						"Description": "CVE-2022-23307"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2041967",
						"Description": "bz#2041967: CVE-2022-23307 log4j: Unsafe deserialization flaw in Chainsaw log viewer"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2022-01-18T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		}
	],
	"producttree": {}
}
